src/main/java/Main/Casillero.java

```diff
@@ -35,7 +32,7 @@ public int deJugador() {
     public Casillero obtenerSiguienteEnDireccion(String direccion) {
         Direccion direccionA = new Direccion(direccion);
         Posicion posicionNueva = this.posicion().sumar(direccionA.posicionRelativa());
-        Casillero nuevoCasillero = Juego.getInstance().tablero().obtenerCasilleroLibre(posicionNueva);
+        Casillero nuevoCasillero = Juego.getInstance().tablero().obtenerCasilleroEnPosicion(posicionNueva);
```

Comment:
Se viola la ley de Demeter.

Reply:
Esto fue un Fix a ultima hora para que el casillero le mande mensajes a tablero. Seria mejor que el tablero sea un atributo de Casillero, no?

Reply:
La primera alternativa sería mediante delegación, es decir, que esa línea quede así:
```java
Casillero nuevoCasillero = Juego.getInstance().obtenerCasilleroEnPosicion(posicionNueva);
```
O sea, la instancia de Juego ahora pasaría a entender el mensaje `obtenerCasilleroEnPosicion(Posicion) : Casillero` cuya implementación consistirá in delegarle la responsabilidad al tablero. De esta manera, la dependencia del casillero es con la instancia de Juego y no con la de Tablero. El problema es que como está ahora el casillero conoce al Juego solamente para poder luego conocer al Tablero utilizando al Juego como intermediario.

En una futura iteración se podría evaluar si conviene o no tener al tablero como atributo del casillero. Suena peligroso porque el tablero ya conoce los casilleros. Otra alternativa sería hacer que el Singleton esté en el tablero y no en el Juego. De todas formas ya será debate para otro PR.